### ISSUE TITLE
ember-*** packages include the `tmp` dir.

### ISSUE THREAD

**Issue by ChALkeR** [emberjs/ember.js#11316]
*Disclaimer: I do not use the package, this was found during automated npm packages downloading.*

`ember-***` npm packages commonly include `tmp` (and `.idea`) directories.
`tmp` directories have sizes from 0 MiB to **621 MiB** per package. They include duplicate copies of some scripts, and I suspect that are not actually needed in the distribution. I suspect that these directories grow over time (from release to release).

This seems to be a common problem for all `ember-***` packages, hence reporting here.

List of packages (may be incomplete) with the `tmp` directory size in bytes:
```
651622411       ember-shepherd-0.1.1.tgz/tmp
547791776       ember-accessible-icon-0.0.3.tgz/tmp
238722262       ember-cli-flot-0.0.3.tgz/tmp
184765255       emberx-select-1.1.2.tgz/tmp
180687876       ember-cli-trackjs-0.1.3.tgz/tmp
122985898       ember-cli-sync-for-each-1.0.8.tgz/tmp
84743005        ember-i18next-0.2.0.tgz/tmp
84043588        ember-cli-skycon-0.0.3.tgz/tmp
84022828        ember-cli-jquery-classy-loader-0.0.2.tgz/tmp
78177959        ember-cli-auto-register-0.2.1.tgz/tmp
57715286        emberx-slider-1.0.3.tgz/tmp
55423165        ember-cli-filter-by-query-1.0.2.tgz/tmp
47370328        ember-cli-copyable-0.9.3.tgz/tmp
38571430        ember-credit-cards-0.0.2.tgz/tmp
36264369        ember-foundation-apps-0.0.3.tgz/tmp
35792350        ember-cli-panels-0.0.6.tgz/tmp
31687314        ember-cli-uncharted-describe-models-0.0.6.tgz/tmp
26192768        ember-cli-rx-0.2.5.tgz/tmp
23656533        ember-cli-poly-component-0.0.5.tgz/tmp
23162805        ember-semantic-analysis-0.0.2.tgz/tmp
19926529        ember-spin-button-0.1.4.tgz/tmp
19826585        ember-cli-static-pages-0.0.5.tgz/tmp
19789187        ember-state-services-0.0.4.tgz/tmp
18049399        ember-cli-array-sequence-1.1.1.tgz/tmp
17465260        ember-poetic-crud-mixins-1.0.1.tgz/tmp
17106891        ember-easy-form-extensions-0.2.10.tgz/tmp
16998849        ember-cli-injection-0.2.1.tgz/tmp
15758956        ember-cli-mapkit-0.1.0.tgz/tmp
15514798        ember-component-css-0.1.2.tgz/tmp
15214810        ember-cli-showdown-2.1.4.tgz/tmp
15105683        ember-cli-siren-adapter-1.0.0.tgz/tmp
13754954        ember-cli-accounting-0.1.1.tgz/tmp
12865229        ember-modals-0.4.0.tgz/tmp
12512120        ember-authr-0.0.1.tgz/tmp
12307605        ember-cli-uncharted-ajax-0.0.6.tgz/tmp
12150522        ember-cli-conditional-compile-0.0.4.tgz/tmp
11882944        ember-pikaday-0.6.0.tgz/tmp
11334130        ember-redirect-1.1.0.tgz/tmp
10156373        ember-breadcrumbs-0.1.6.tgz/tmp
9557614         ember-cli-scrolling-content-1.1.1.tgz/tmp
9393431         ember-cli-firebase-hosting-0.1.2.tgz/tmp
9280354         ember-cli-chosen-0.0.3.tgz/tmp
9125369         ember-timetree-2.0.1.tgz/tmp
8523072         ember-cli-stained-by-children-0.4.0.tgz/tmp
8011447         ember-cli-file-creator-0.1.0.tgz/tmp
7598409         ember-data-fixture-adapter-1.0.0.tgz/tmp
7245142         ember-cli-filepicker-0.1.3.tgz/tmp
7052179         ember-cli-bugsnag-0.0.9.tgz/tmp
6405953         ember-cli-chartist-0.2.7.tgz/tmp
5765901         ember-aviary-0.0.7.tgz/tmp
5521195         ember-magic-man-0.0.9.tgz/tmp
5448914         ember-cli-table-0.0.2.tgz/tmp
4956285         ember-slider-0.0.3.tgz/tmp
4208830         ember-css-routes-0.0.2.tgz/tmp
2172055         ember-fsm-0.4.2.tgz/tmp
451008          emberui-0.6.0.tgz/tmp
379234          ember-cli-ember-table-0.1.1.tgz/tmp
200533          ember-cpm-1.3.2.tgz/tmp
137843          ember-cli-ghost-app-0.1.0.tgz/tmp
8689            ember-loader-0.0.20.tgz/tmp
8201            ember-cli-gravatar-3.0.1.tgz/tmp
4105            ember-try-0.0.5.tgz/tmp
4105            ember-percentages-1.0.2.tgz/tmp
4105            ember-list-view-0.0.6.tgz/tmp
4105            ember-linkify-1.2.1.tgz/tmp
4105            ember-keen-tracking-0.0.7.tgz/tmp
4105            ember-keen-querying-0.0.7.tgz/tmp
4105            ember-instance-scope-0.0.2.tgz/tmp
4105            ember-get-helper-0.0.5.tgz/tmp
4105            ember-gauge-0.0.6.tgz/tmp
4105            ember-feature-flags-1.0.0.tgz/tmp
4105            ember-drop-0.0.1.tgz/tmp
4105            ember-deploy-couchbase-0.1.1.tgz/tmp
4105            ember-cli-summernote-0.1.15.tgz/tmp
4105            ember-cli-styleless-tokenfield-0.0.4.tgz/tmp
4105            ember-cli-pure-1.0.0.tgz/tmp
4105            ember-cli-placeholdit-0.0.3.tgz/tmp
4105            ember-cli-make-concoction-0.0.5.tgz/tmp
4105            ember-cli-google-street-view-0.1.2.tgz/tmp
4105            ember-cli-deploy-rsync-0.0.1.tgz/tmp
4105            ember-cli-dates-1.1.1.tgz/tmp
4105            ember-cli-crudtable-0.4.2.tgz/tmp
4105            ember-cli-cloudinary-images-0.1.0.tgz/tmp
4105            ember-cli-bootstrap-switch-0.1.7.tgz/tmp
4105            ember-cli-autoprefixer-0.4.0.tgz/tmp
4105            ember-cli-alcatraz-client-0.1.1.tgz/tmp
4105            ember-cli-active-link-wrapper-0.0.2.tgz/tmp
```

I found `tmp` dir in 87 packages and `.idea` dir in 64 packages. There could be more.

**Comment by ChALkeR** [comment]
I reported that to three packages already. If you want to fix that per-package, you could file appropriate issues yourself.

**Comment by ChALkeR** [comment]
Migrated to https://github.com/ember-cli/ember-cli/issues/4199 after IRC discussion.

**Comment by rwjblue** [comment]
This was fixed in https://github.com/ember-cli/ember-cli/pull/3539, and has been included in versions since ember-cli@0.2.1.  Once these packages update to > 0.2.1 (unless they opt out of the upgrade) they will properly ignore `tmp` and `dist.
